rke/templates/coredns.go

```diff
@@ -399,11 +407,15 @@ metadata:
     k8s-app: kube-dns
     kubernetes.io/name: "CoreDNS"
 spec:
-  replicas: 1
   strategy:
+{{if .UpdateStrategy}}
+{{ toYaml .UpdateStrategy | indent 4}}
+{{else}}
     type: RollingUpdate
     rollingUpdate:
-      maxUnavailable: 1
+      maxUnavailable: 25%
```

Comment:
I think we should be passing our defaults from RKE rather than trying to define defaults in the template here